tests/EmailVerificationTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+use App\API\V1\Repositories\EmailVerificationRepository;
+use App\API\V1\Repositories\RoleRepository;
+use App\API\V1\Repositories\UserRepository;
+use Faker\Factory;
+use App\API\V1\Entities\User;
+use TempestTools\Scribe\PHPUnit\CrudTestBaseAbstract;
+
+class EmailVerificationTest extends CrudTestBaseAbstract
+{
+    protected $password = '441520435a0a2dac143af05b55f4b751';
+
+
+    /**
+     * Test that acl middle ware works in allowing some one to access an end point
+     * @group aclMiddleware
+     * @return void
+     * @throws Exception
+     */
+    public function testGuestAccess():void
+    {
+        $generator = Factory::create();
+        $em = $this->em();
+        $conn = $em->getConnection();
+        $conn->beginTransaction();
+        try {
+            $email = $generator->safeEmail;
+            $password = $this->password;
+            /** Register user via the guest endpoint **/
+            $response = $this->json(
+                'POST', '/contexts/guest/users',
+                [
+                    "params" => [
+                        "name" => $generator->name,
+                        "email" => $email,
+                        "password" => $password,
+                        "job" => $generator->jobTitle,
+                        "address" => $generator->address,
+                        "locale" => "en"
+                    ],
+                    "options" => [
+                        "email" => false,
+                        "g-recaptcha-response-omit" => env('GOOGLE_RECAPTCHA_SKIP_CODE') /** skipping recaptcha with a key **/
```

Comment:
@jerazo just to clarify, we can disable the skip code if we want too correct? So if we decide it's a vulnerability we can switch it off from the env?

Reply:
Is there any danger someone could get the skip code (by getting into our environment) and then bypass our captcha? @jerazo just trying to make sure we can disable the ability to use a skip code at all if we want to turn off that functionality.